Update README.md (#1)
Cabeda/Cabeda: README.md

<img align="left" width="50" height="50" src="https://github.githubassets.com/images/modules/logos_page/GitHub-Mark.png" alt="Github Icon">

## Welcome to my Github Profile 

#### My name is José Cabeda and I'm a developer from Porto, Portugal 🇵🇹
### For more information visit my personal website https://cabeda.me/
##

> 🏢 I'm currently work at Talkdesk as a Data Enginner.

Author: ```suggestion
🏢 I'm currently working at Talkdesk as a Data Engineer.
```

<img width="20" height="25" src="https://img2.pngio.com/download-icon-css-3-svg-eps-png-psd-ai-logo-vector-color-free-el-css-logo-png-1141_1600.png" alt="CSS Icon">
<img width="25" height="25" src="https://upload.wikimedia.org/wikipedia/commons/thumb/a/a7/React-icon.svg/1200px-React-icon.svg.png" alt="React Icon">

⚡ Fun facts about me, I love reading and learning new things on the Internet 🌐

##

### How to reach me 📫

<img align="left" width="25" height="25" src="https://www.vectorico.com/wp-content/uploads/2018/02/LinkedIn-Icon-squircle.png" alt="Linkedin Icon"> https://www.linkedin.com/in/jecabeda/

<img align="left" width="25" height="25" src="https://github.githubassets.com/images/modules/logos_page/GitHub-Mark.png" alt="Github Icon"> https://github.com/Cabeda/

<img align="left" width="25" height="25" src="https://image.similarpng.com/very-thumbnail/2020/06/Logo-Twitter-icon-transparent-PNG.png" alt="Twitter Icon"> https://twitter.com/Jumpingtolife

<img align="left" width="25" height="25" src="https://cdn4.iconfinder.com/data/icons/free-colorful-icons/360/gmail.png" alt="Twitter Icon"> 

> jecabeda@gmail.com

Author: ```suggestion
jecabeda at gmail.com
```

> jecabeda@gmail.com

Reviewer: ```suggestion
jecabeda@gmail.com
```
